app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/LoginResponse.kt

```diff
@@ -0,0 +1,31 @@
+package org.fossasia.susi.ai.rest.responses.susi
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+/**
+ * <h1>POJO class to parse retrofit response from login endpoint susi client.</h1>
+
+ * Created by saurabh on 12/10/16.
+ */
+
+class LoginResponse {
+
+    @SerializedName("message")
+    @Expose
+    val message: String? = null
+
+    @SerializedName("session")
+    @Expose
+    val session: Session? = null
+
+    @SerializedName("valid_seconds")
+    @Expose
+    val validSeconds: Long = 0
+
+    @SerializedName("access_token")
+    @Expose
+    var accessToken: String? = null
+        internal set
```

Comment:
internal set is one of the modifiers like that of public, private and protetected. It can be used to access the variable at same package level. 

Refer to this 
https://kotlinlang.org/docs/reference/visibility-modifiers.html